backend/templates/README.md

```diff
@@ -0,0 +1,8 @@
+# Templates Using templ
```

Comment:
maybe there's not a better design but is there a way to not have individualized folders for each of the templates?

Reply:
the reason it's organized like that is so ech template is a package, so we can call`welcome.Index(...)`. I could reorg so they're all in an emails package and it would be `emails.Welcome(...)`

Reply:
just reorged